hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestDataNodeMetrics.java

```diff
@@ -816,4 +817,28 @@ public Boolean get() {
       }, 100, 10000);
     }
   }
+
+  @Test
+  public void testDataNodeDatasetLockMetrics() throws IOException {
+    Configuration conf = new HdfsConfiguration();
+    MiniDFSCluster cluster = new MiniDFSCluster.Builder(conf).build();
+    try {
```

Comment:
It's best to use `try-with-resources` here.

Reply:
@tomscut Thanks a lot Sir,  have fixed.